terraform/utils.go

```diff
@@ -38,22 +41,45 @@ func tfConfigList(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateDat
 	// will never match the requested value.
 	quals := d.EqualsQuals
 	if quals["path"] != nil {
-		d.StreamListItem(ctx, filePath{Path: quals["path"].GetStringValue()})
+
+		path := d.EqualsQualString("path")
+
+		fileContent, err := os.ReadFile(path)
```

Comment:
updated.

Reply:
@cbruno10, we need to do the `isTerraformPlan` check here. 

The main reason is if we pass a file path using the `path` qual, the plugin is not aware of what kind of file it is, whether it is a configuration file or a terraform plan.

Reply:
@cbruno10, we need to do the `isTerraformPlan` check here. 

The main reason is if we pass a file path using the `path` qual, the plugin is unaware of its kind of file, whether it is a configuration file or a terraform plan.

Generally, while we are getting the configured file paths from the `terraform.spc` file, we know that all the TF plan files are provided using the `planFilePaths` argument. but when we use the qual to pass the file path, we don't know the file type.

Due to that reason, when we are reading the file path from the qual, we are checking for a few specific common properties in the file related to the TF plan, and if those are present in the file, we set the `IsTFPlanFilePath` as true, which we check in the table.

Reply:
@bigdatasourav, the changes that you have made are partially correct. We also need to change the other tables since if we pass a file path using the `path` qual, the plugin is unaware of its kind of file, whether it is a configuration file or a terraform plan.

For example, if we query any table other than a resource with a path qual that contains a Tf plan file path, the query will fail since the TF plan parsing is only supported in the resource table.

Reply:
I have also updated the other tables.